.github/workflows/build_wheels_linux.yml

```diff
@@ -18,34 +18,40 @@ on:
 
 jobs:
   Build:
-    runs-on: ubuntu-22.04
+    runs-on: ${{ matrix.platform == 'aarch64' && 'opencv-cn-lin-arm64' || 'ubuntu-22.04' }}
     defaults:
       run:
         shell: bash
     strategy:
       fail-fast: false
-      matrix:
-        python-version: ['3.9']
-        platform: [x64]
-        with_contrib: [0, 1]
-        without_gui: [0, 1]
+      matrix: 
+        python-version: ['3.9'] 
+        platform: [x64, aarch64] 
+        manylinux: [2014, 2_28] 
+        with_contrib: [0, 1] 
+        without_gui: [0, 1] 
         build_sdist: [0]
+        exclude:
+          - platform: aarch64
+            manylinux: 2014
+
     env:
       ACTIONS_ALLOW_UNSECURE_COMMANDS: true
       REPO_DIR: .
       PROJECT_SPEC: opencv-python
       MB_PYTHON_VERSION: ${{ matrix.python-version }}
       TRAVIS_PYTHON_VERSION: ${{ matrix.python-version }}
-      MB_ML_VER: 2014
+      MB_ML_VER: ${{ matrix.manylinux }}
       TRAVIS_BUILD_DIR: ${{ github.workspace }}
       CONFIG_PATH: travis_config.sh
-      DOCKER_IMAGE: quay.io/opencv-ci/opencv-python-manylinux2014-x86-64:20250630
+      DOCKER_IMAGE: ${{ matrix.manylinux == '2014' && (matrix.platform == 'x64' && 'quay.io/opencv-ci/opencv-python-manylinux2014-x86-64:20250630') || '' || (matrix.platform == 'x64' && 'quay.io/opencv-ci/opencv-python-manylinux_2_28-x86-64:20251013') || 'quay.io/opencv-ci/opencv-python-manylinux_2_28-aarch64:20251013' }}
       USE_CCACHE: 0
       UNICODE_WIDTH: 32
-      PLAT: x86_64
+      PLAT: ${{ matrix.platform == 'x64' && 'x86_64' || 'aarch64' }}
```

Comment:
Looks strange.

You can use `platform: [x86-64, aarch64]` in matrix and set docker image as `DOCKER_IMAGE: quay.io/opencv-ci/opencv-python-manylinux_{{ matrix.manylinux }}-{{ matrix.platform }}:20251013`